src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/NewBlogPostStoringActor.java

```diff
@@ -25,11 +26,20 @@
     public NewBlogPostStoringActor(BlogPostRepository blogPostRepository) {
         receive(ReceiveBuilder.match(RssEntryWithAuthor.class,
             rssEntry -> {
-                BlogPost blogPost = blogPostRepository
-                    .findByUrl(rssEntry.getRssEntry().getLink())
-                    .orElseGet(() -> createBlogPost(rssEntry));
-                updateDescription(blogPost, rssEntry.getRssEntry().getDescription());
-                blogPostRepository.save(blogPost);
+
+                if (Validators.isUrlValid(rssEntry.getRssEntry().getLink())) {
+                    BlogPost blogPost = blogPostRepository
+                        .findByUrl(rssEntry.getRssEntry().getLink())
+                        .orElseGet(() -> createBlogPost(rssEntry));
+                    updateDescription(blogPost, rssEntry.getRssEntry().getDescription());
+                    blogPostRepository.save(blogPost);
```

Comment:
\[Checkstyle\] WARNING: 'if' child have incorrect indentation level 20, expected level should be one of the following: 14, 16, 18\.